### ISSUE TITLE
Adding a read-Only Client Certificate on a one-node cluster

### ISSUE THREAD

**Issue by ojasp** [Azure/service-fabric-issues#1588]
I tried adding a new Read-Only Client certificate and was presented with the following error:

> Failed to submit updates for certificate
> Failed to submit updates to 'read-only client certificate' for XXXXX

I then also went to resources.azure.com and tried to change the thumbprint of one of the existing admin certificates to the thumbprint of the new certificate that I created, but was presented with a Yellow triangle and nothing was changed. Any other way to add a read-only client certificate to our existing cluster?

**Comment by masnider** [comment]
What steps did you follow? 

In these docs, we try to lay out the process. https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-security-update-certs-azure If you could step through these instructions and let us know where you have problems, that'd be great.

**Comment by ojasp** [comment]
@masnider I opened a ticket with Azure shortly after creating this issue and I was told that a single node cluster is treated as a test cluster and does not have the capability to add another certificate once the cluster is created. If I wanted to add another certificate then I will have to redeploy the entire cluster, which while not ideal, was something I ended up doing.

**Comment by tijoytom** [comment]
@ojasp Yes, you are right.  Upgrade is not supported for single-node clusters
